lib/evss/auth_headers.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
-require 'evss/base_headers'
```

Comment:
Instead of editing the existing file, can we create a new file for these changes? I think we will want to have them both in place so we can toggle between them during the migration process.

Reply:
This comment goes for all the files in this PR just FYI

Reply:
FOr example, create a file named `lib/lighthouse/auth_headers.rb` and so on